packages/app-store/routing-forms/pages/form-edit/[...appPages].tsx

```diff
@@ -85,30 +87,70 @@ function Field({
   };
   appUrl: string;
 }) {
-  const [identifier, _setIdentifier] = useState(hookForm.getValues(`${hookFieldNamespace}.identifier`));
   const { t } = useLocale();
 
-  const setUserChangedIdentifier = (val: string) => {
-    _setIdentifier(val);
-    // Also, update the form identifier so tha it can be persisted
-    hookForm.setValue(`${hookFieldNamespace}.identifier`, val);
+  const [options, setOptions] = useState<SelectOption[]>([
+    { placeholder: "< 10", value: "" },
+    { placeholder: "10-100", value: "" },
+    { placeholder: "100-500", value: "" },
+    { placeholder: "> 500", value: "" },
+  ]);
+
+  const handleRemoveOptions = (index: number) => {
+    const updatedOptions = options.filter((_, i) => i !== index);
+    setOptions(updatedOptions);
+    updateSelectText(updatedOptions);
   };
 
-  const label = hookForm.watch(`${hookFieldNamespace}.label`);
+  const handleAddOptions = () => {
+    setOptions((prevState) => [
+      ...prevState,
+      {
+        placeholder: "New Option",
+        value: "",
+      },
+    ]);
+  };
 
   useEffect(() => {
-    if (!hookForm.getValues(`${hookFieldNamespace}.identifier`)) {
-      _setIdentifier(label);
+    const originalValues = hookForm.getValues(`${hookFieldNamespace}.selectText`);
+    if (originalValues) {
+      const values: SelectOption[] = originalValues.split("\n").map((fieldValue) => ({
+        value: fieldValue,
+        placeholder: "",
+      }));
+      setOptions(values);
     }
-  }, [label, hookFieldNamespace, hookForm]);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
   const router = hookForm.getValues(`${hookFieldNamespace}.router`);
   const routerField = hookForm.getValues(`${hookFieldNamespace}.routerField`);
+
+  const handleChange = (e: React.ChangeEvent<HTMLInputElement>, optionIndex: number) => {
+    const updatedOptions = options.map((opt, index) => ({
+      ...opt,
+      ...(index === optionIndex ? { value: e.target.value } : {}),
+    }));
+    setOptions(updatedOptions);
+    updateSelectText(updatedOptions);
+  };
+
+  const updateSelectText = (updatedOptions: SelectOption[]) => {
+    hookForm.setValue(
+      `${hookFieldNamespace}.selectText`,
+      updatedOptions
+        .filter((opt) => opt.value)
+        .map((opt) => opt.value)
+        .join("\n")
+    );
+  };
   return (
     <div
       data-testid="field"
-      className="group mb-4 flex w-full items-center justify-between ltr:mr-2 rtl:ml-2">
+      className="bg-default group mb-4 flex w-full items-center justify-between ltr:mr-2 rtl:ml-2">
       <FormCard
-        label={label || `Field ${fieldIndex + 1}`}
+        label={`Form ${fieldIndex + 1}`}
```

Comment:
This is supposed to be called Field only. The tab that shows it has name Form, so it can't have multiple forms. These are fields.